README.md

```diff
@@ -1,29 +1,35 @@
-# Vipps order management api v1
+# Vipps Order Management API v1
 
 💥 Release Candidate! This is unfinished work and subject to change. 💥
 
-The Order Management API allows merchants to send rich receipt information to existing Vipps transaction. This information is shown to the customer in the app in their order history.
-This functionality is available for both recurring and direct payments, but not for passthrough payments.
+The Vipps Order Management API allows merchants to send rich receipt information to existing Vipps transactions. This information is shown to the customer in the app in their order history.
+This functionality is available for both
+[recurring](https://github.com/vippsas/vipps-ecom-api)
+and
+[direct payments](https://github.com/vippsas/vipps-recurring-api),
+but not for
+[passthrough payments](https://github.com/vippsas/vipps-psp-api).
```

Comment:
passthrough isn't allowed to use this

Reply:
Renaming: Ok, but can the swagger file be placed in the "docs" directory, so Swagger UI and ReDoc works?

I see I copied and pasted wrong, will fix.